runtime/browser/android/cookie_manager.cc

```diff
@@ -7,28 +7,43 @@
 #include <string>
 
 #include "android_webview/browser/scoped_allow_wait_for_legacy_web_view_api.h"
```

Comment:
I think it will be confused to copy a file that named "web_view" into xwalk, and no benefit.
@axinging @fujunwei @XiaosongWei  any comments on this?